osc-collab.py

```diff
@@ -3620,7 +3620,8 @@ def _collab_add_config_option(section, key, value):
         elif line[0] == '[' and in_section and not added:
             if not empty_line:
                 os.write(fdout, '\n')
```

Comment:
I guess this should be changed to `b'\n'`. And there may be other occurrences of this in the code. Can you check it out?